svthalia/concrexit
# Timeout to registrations and renewals

Closes #3606.




### Summary

Add a task that checks if there are registrations that have been open for more then a month. Send a reminder mail to the board if this is the case. Delete the registration if the registration is open for more than 3 months. 

### How to test

1. Add an open registration that has been open for more than a month
2. Exercute the task
3. See the mail
4. Check if the registration is deleted after setting the created_at field on a date more than 3 months in the past
